packages/flutter_tools/test/daemon_test.dart

```
@@ -195,7 +182,7 @@ void main() {
       commands.close();
     });
 
-    _testUsingContext('daemon.stop', () async {
+    testUsingContext('daemon.stop', () async {
```

"app.stop without appId should report an error"